src/main/java/com/laserfiche/api/client/httphandlers/HttpRequestHandler.java

```diff
@@ -17,4 +17,10 @@ public interface HttpRequestHandler {
      * @return True if the request should be retried.
      */
     CompletableFuture<Boolean> afterSendAsync(Response response);
+
+    /**
+     * Overrides close() in AutoCloseable as no meaningful exception can be handled.
+     */
+    @Override
+    void close();
```

Comment:
as this is an interface, we don't need to declare close() here.

Reply:
Thanks. See the above reply.